src/screens/SubTags/SubTags.spec.tsx

```diff
@@ -22,6 +21,8 @@ import i18n from 'utils/i18nForTest';
 import SubTags from './SubTags';
 import { MOCKS, MOCKS_ERROR_SUB_TAGS } from './SubTagsMocks';
 import { InMemoryCache, type ApolloLink } from '@apollo/client';
+import { vi, beforeEach, afterEach, expect, it } from 'vitest';
+import '@testing-library/jest-dom/vitest';
```

Comment:
Why was this imported if we are migrating from jest and it doesn't appear to be used?

Reply:
Sure, I am removing this and will push very soon.
Thanks for guiding.